### PR TITLE
Shoshi/home page

### PR DESCRIPTION
**Pull request by Shoshana-Yudelevitz**
הוספתי קומפוננטה של עמוד בית -homePageComponent
הוספתי לזה עמוד עיצוב -homePage בתיקית STYLES

### CODE REVIEW

**Review by dvory-git — APPROVED** [review]

File: client-side/src/components/homeComponent/homePageComponent.jsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import '../../styles/homePage.scss';
+
+const HomePage = () => {
+  return (
+    <div className="homepage">
+      <main>
+        <section className="feature even">
+          <div className="content">
+            <div className="text">
+              <h2>Enhance Your Browsing Experience with Our Chrome Add-on</h2>
```
Comment (e171271) [inline client-side/src/components/homeComponent/homePageComponent.jsx:14]: please move all texts to constants file - do it in all your code

**Review by e171271 — CHANGES_REQUESTED** [review]

File: client-side/src/axios/middleware.js [hunk 2]
```diff
@@ -3,12 +3,14 @@ import axios from 'axios';
 const url = process.env.REACT_APP_SERVER_URL;
 
 export async function handleGet(path) {
+    console.log(url)
```
Comment (e171271) [inline client-side/src/axios/middleware.js:6]: please remove all occurrences of console log from your code

File: client-side/src/components/profileComponent.jsx
Comment (e171271) [inline client-side/src/components/profileComponent.jsx:1]: why did you remove this component? is it a mistake or not?

File: client-side/src/components/profileComponents/ProfileActions.jsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Tooltip, Button } from '@mui/material';
+import { TOOLTIP_TEXTS, BUTTON_LABELS } from '../../constants/profileConstants.js';
+
+export default function ProfileActions({ handleSave, handleClose, handleDelete }) {
```
Comment (e171271) [inline client-side/src/components/profileComponents/ProfileActions.jsx:5]: please add default values for each prop that not marked as required, also please add prop-type definition

File: client-side/src/components/profileComponents/ProfileActions.jsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    return (
+        <>
+            <Tooltip title={TOOLTIP_TEXTS.CANCEL}>
+                <Button sx={{ color: ' rgb(103, 252, 210) ' }} onClick={handleClose}>
```
Comment (e171271) [inline client-side/src/components/profileComponents/ProfileActions.jsx:9]: colors should be in scss file - do it in all your code

File: client-side/src/components/profileComponents/ProfileForm.jsx [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+//                                 onChange={(e) => handleFieldChange(e, setFormData)}
+//                                 options={SELECT_OPTIONS.STATUS_BLOCKED_SITES}
+//                                 widthOfSelect='100%'
+//                             />
+//                         </Tooltip>
+//                     </Box>
+//                 </Grid>
+//             </Grid>
+//         </Box>
+//     );
```
Comment (e171271) [inline client-side/src/components/profileComponents/ProfileForm.jsx:71]: please remove comments from all your code

File: client-side/src/components/profileComponents/ProfileForm.jsx [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+// }
+
+
+import React from 'react';
+import { Grid, Box, Tooltip, FormControl, InputLabel, Select as MuiSelect, MenuItem } from '@mui/material';
+import GenericInput from '../../stories/GenericInput/genericInput.jsx';
+import { INPUT_LABELS, TOOLTIP_TEXTS, SELECT_OPTIONS } from '../../constants/profileConstants.js';
+import { handleFieldChange } from '../../utils/profileUtil.js';
+
+export default function ProfileForm({ formData, setFormData }) {
```
Comment (e171271) [inline client-side/src/components/profileComponents/ProfileForm.jsx:81]: please add default values for each prop that not marked as required, also please add prop-type definition - do it for all components

File: client-side/src/components/profileComponents/ProfileForm.jsx
Comment (e171271) [inline client-side/src/components/profileComponents/ProfileForm.jsx:1]: why profiles changes is related to hompePage development?

**Review by e171271 — CHANGES_REQUESTED** [review]
this branch should have only changes that are connected to homePage development

**Pull request closed without merging**